runelite-client/src/main/java/net/runelite/client/plugins/party/PartyConfig.java

```diff
@@ -74,10 +68,33 @@ default boolean messages()
 	@ConfigItem(
 		keyName = "recolorNames",
 		name = "Recolor names",
-		description = "Recolor stats overlay names based on unique color hash"
+		description = "Recolor party members names based on unique color hash",
+		position = 4
 	)
 	default boolean recolorNames()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "autoOverlay",
+		name = "Auto overlay",
+		description = "Automatically add an overlay with player data when a member joins",
+		position = 5
+	)
+	default boolean autoOverlay()
+	{
+		return false;
```

Comment:
Why is this false? You should preserve the original behaviour

Reply:
Yes I wasn't sure about this. I thought It would duplicate the info by default. I agree the default behavior should be preserved.